1-org/envs/shared/log_sinks.tf

```diff
@@ -38,83 +39,79 @@ resource "random_string" "suffix" {
 }
 
 /******************************************
   Send logs to Storage
 *****************************************/
 
-module "log_export_to_storage" {
-  source                 = "terraform-google-modules/log-export/google"
-  version                = "~> 7.3.0"
-  destination_uri        = module.storage_destination.destination_uri
-  filter                 = local.all_logs_filter
-  log_sink_name          = "sk-c-logging-bkt"
-  parent_resource_id     = local.parent_resource_id
-  parent_resource_type   = local.parent_resource_type
-  include_children       = true
-  unique_writer_identity = true
-}
-
 module "storage_destination" {
-  source                      = "terraform-google-modules/log-export/google//modules/storage"
-  version                     = "~> 7.3.0"
-  project_id                  = module.org_audit_logs.project_id
-  storage_bucket_name         = "bkt-${module.org_audit_logs.project_id}-org-logs-${random_string.suffix.result}"
-  log_sink_writer_identity    = module.log_export_to_storage.writer_identity
-  uniform_bucket_level_access = true
-  location                    = var.log_export_storage_location
-  retention_policy            = var.log_export_storage_retention_policy
-  force_destroy               = var.log_export_storage_force_destroy
-  versioning                  = var.log_export_storage_versioning
+  source = "../../modules/centralized-logging"
+
+  logging_sink_filter            = local.all_logs_filter
+  logging_sink_name              = "sk-c-logging-bkt"
+  resources                      = local.parent_resources
+  resource_type                  = local.parent_resource_type
+  include_children               = true
+  logging_target_type            = "storage"
+  logging_destination_project_id = module.org_audit_logs.project_id
+  logging_target_name            = "bkt-${module.org_audit_logs.project_id}-org-logs-${random_string.suffix.result}"
+  uniform_bucket_level_access    = true
+  logging_location               = var.log_export_storage_location
+  retention_policy               = var.log_export_storage_retention_policy
+  delete_contents_on_destroy     = var.log_export_storage_force_destroy
+  versioning                     = var.log_export_storage_versioning
 }
 
 /******************************************
   Send logs to Pub\Sub
 *****************************************/
 
-module "log_export_to_pubsub" {
-  source                 = "terraform-google-modules/log-export/google"
-  version                = "~> 7.3.0"
-  destination_uri        = module.pubsub_destination.destination_uri
-  filter                 = local.main_logs_filter
-  log_sink_name          = "sk-c-logging-pub"
-  parent_resource_id     = local.parent_resource_id
-  parent_resource_type   = local.parent_resource_type
-  include_children       = true
-  unique_writer_identity = true
+module "pubsub_destination" {
+  source = "../../modules/centralized-logging"
```

Comment:
With centralized-logging, I was hoping we could require only 1 module instantiation. So the user can pass in a list of `logging_target_type` and we setup all those destinations.

Reply:
Done